app/src/main/res/values-v23/themes.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="AppTheme.NoActionBar.Splash" parent="AppTheme.NoActionBar">
```

Comment:
I'm not sure that it's not necessary or not 🤔 
It's same with `values-v21/themes.xml`, right?